Interrupter Software/Makefile

```diff
@@ -0,0 +1,36 @@
+FLAGS = -Os -Wall -mmcu=atmega328p -c
+
+DRIVERS_SRC = drivers
+DRIVERS_INC = drivers/includes
+FUNC_INC = functionalities/includes
+FUNC_SRC = functionalities
+
+compile: main.o drivers functionalities
+	avr-gcc main.o ${DRIVERS_SRC}/eepromdvr.o ${DRIVERS_SRC}/uartdvr.o ${FUNC_SRC}/music.o ${FUNC_SRC}/upload.o ${FUNC_SRC}/command.o -o main.elf
+	avr-objcopy -O ihex main.elf main.hex
+	rm main.elf main.o ${DRIVERS_SRC}/uartdvr.o ${DRIVERS_SRC}/eepromdvr.o ${FUNC_SRC}/music.o ${FUNC_SRC}/upload.o ${FUNC_SRC}/command.o 
+
```

Comment:
Why you delete object files in compile time, instead of the user can choose to delete them himself. Maybe it's a better solution to delete them in a new tag such as "clean" or "clear". And what is your execution file in the end?

Reply:
There is no point of leaving object files in the project's directory. Since the code is open source if the user decides that he wants to keep the files, he can change the makefile and add the 'clean' option. The 'Clean' option is used separately when developing because after the first compilation if the user changes one file, the others wont compile resulting in a much faster compilation. Since the user is meant to compile it once there is no point of having so many object files which are only useful for multiple compilations.